sample_app/GSDML-V2.4-rtlabs-IODevice-20201211.xml

```diff
@@ -176,11 +176,22 @@
                </VirtualSubmoduleList>
             </ModuleItem>
          </ModuleList>
+         <LogBookEntryList>
+            <LogBookEntryItem Status="2130510">
```

Comment:
I do not get where this comes from. Maybe a comment should be added

Reply:
Done